docs/src/index.ejs

```diff
@@ -124,6 +124,20 @@
     .hidden {
       display: none;
     }
+
+    /*
+      TODO: this could solve the isFromKeyboard and onFocus complication.
+            Need to work out how we integrate this static style into style functions.
+
+    [data-whatinput="mouse"] *:focus {
```

Comment:
that would be good. when we checked a year back, fela did not allow these kind of selectors. But with unified focus indicator, it might be easier to have just global style that has nothing to do with fela. We will need to try and see. It would be also good to have screener tests of focused state for all components before (if) we decide to try this

Reply:
Moved to comment https://github.com/stardust-ui/react/issues/749#issuecomment-507267679